test/jquery.ime.test.js

```diff
@@ -241,7 +241,7 @@
 			$input.ime();
 			$input.focus();
 			ime = $input.data( 'ime' );
-			QUnit.expect( 1 );
+			assert.expect( 1 );
```

Comment:
This fails in QUnit 2 because this is using the global singleton variable `assert` instead of the contextual one passed to `QUnit.test`. Be sure to add it to the callback signature :-). This isn't even deprecated as so much it just happened to work in QUnit 1.x as implementation detail for making the global methods `ok()` and `equal()` work.